src/components/dialogs/reactive-limits/reactive-capability-curve/reactive-capability-curve-table.jsx

```diff
@@ -27,6 +28,19 @@ export const ReactiveCapabilityCurveTable = ({
 }) => {
     const { fields: rows, insert, remove } = useFieldArray({ name: `${id}` });
 
+    // Respect minimum size (2 rows)
+    useEffect(() => {
+        if (rows.length < MIN_LENGTH) {
+            for (let i = 0; i < MIN_LENGTH - rows.length; i++) {
+                rows.push({
```

Comment:
Argument type {p: null, minQ: null, maxQ: null} is not assignable to parameter type FieldArrayWithId<FieldValues, string, "id">

useFieldArray expects the id field and the objects that you are trying to add do not include this id

To fix this error, i suggest to use the insert function like 

`insert(rows.length, {
                [MAX_Q]: null,
                [MIN_Q]: null,
                [P]: null,
            });`

Reply:
modified